www/shared/utils/google-tag-manager/initialize-tag-manager.test.js

```diff
@@ -0,0 +1,31 @@
+const initializeMock = jest.fn();
+
+jest.doMock('react-gtm-module', () => ({ initialize: initializeMock }));
+
+beforeEach(() => {
+    process.env.GTM_TRACKING_ID = 'TESTE-123123';
+});
+
+afterEach(() => {
+    jest.resetModules();
+    jest.clearAllMocks();
+    jest.unmock('ganalytics');
+});
+
+it('should initialize gtm module if tracking id is provided', () => {
+    const initializeTagManager = require('./initialize-tag-manager').default;
+
+    initializeTagManager();
+
+    expect(initializeMock).toHaveBeenCalled();
```

Comment:
We should assert that the `TagManager`'s initialization function is called with the appropriate arguments, namely the `gtmId` at the very least. Otherwise, how we the know the we're calling it correctly?